src/searcher_frame_document.rs

```diff
@@ -56,28 +31,41 @@ impl SearchResult {
     }
 
     #[getter]
-    fn unique_docs(&self, py: Python) -> PyResult<Vec<u64>> {
+    fn unique_docs(&self, py: Python) -> PyResult<BTreeSet<u64>> {
```

Comment:
Now that that the memleak is fixed in pyo3==0.19.1, we don't need to wrap in vecs any more.  This, and the following two changes is the revert of the workaround.